tests/python_tests/ov_genai_test_utils.py

```diff
@@ -13,7 +13,6 @@
 import shutil
 import json
 
-
```

Comment:
Put it back :) https://peps.python.org/pep-0008/ recommends to have two empty lines.